src/utils/query-actions.js

```diff
@@ -54,6 +54,22 @@ const _fetch = async (endpoint, callback, options = {}) => {
         })
         .catch(fetchErrorHandler);
 }
+
+const _fetchPublic = async (endpoint, callback, options = {}) => {
+    return fetch(buildAPIBaseUrl(endpoint.toString()), options)
```

Comment:
this is duplicating code of the original _fetch function
perhaps the _fetch function could use this one and just add the access token ?